src/frontend/main/translations.ts

```diff
@@ -0,0 +1,144 @@
+export default {
+  en: {
+    symptomLabels: {
+      corona_infection_suspicion: 'Corona infection suspicion',
+      corona_suspicion: 'Corona suspicion',
+      cough: 'Cough',
+      fever: 'Fever',
+      breathing_difficulties: 'Breathing difficulties',
+      muscle_pain: 'Muscle pain',
+      headache: 'Headache',
+      sore_throat: 'Sore throat',
+      rhinitis: 'Rhinitis',
+      stomach_issues: 'Stomache issues',
+      sensory_issues: 'Sensory issues',
+      loss_smell_or_taste: 'Loss of smell or taste',
+    },
+    main: {
+      languageSelector: 'Kieli',
+      cityInformation: 'City information',
+      openOrCloseMenu: 'Open/Close menu',
+      close: 'Close',
+      navigation: 'Navigation',
+      allMunicipalities: 'All municipalities',
+      allOfFinland: 'All of Finland',
+      chooseMunicipality: 'Choose the municipality',
```

Comment:
I think a better text here would be "Choose a municipality"? When you think of the context. Though the whole title will probably change soon if that view will become the dashboard.

Reply:
Agree that `Choose a municipality` is more suitable contextually :+1: